fineract-provider/src/main/java/org/apache/fineract/commands/service/CommandWrapperBuilder.java

```diff
@@ -3713,8 +3713,17 @@ public CommandWrapperBuilder loanPayOfLoan(Long loanId) {
     public CommandWrapperBuilder addBusinessDetail(final long clientId) {
         this.actionName = "CREATE_CLIENTBUSINESSDETAIL";
         this.entityName = "CLIENT";
-        this.href = "/clients/" + clientId + "/addbusinessdetail";
+        this.href = "/clients/" + clientId + "/businessDetail";
         this.clientId = clientId;
         return this;
     }
+
+    public CommandWrapperBuilder deleteBusinessDetail(final long clientId, final long businessDetailId) {
+        this.actionName = "DELETE_CLIENTBUSINESSDETAIL";
+        this.entityName = "CLIENT";
```

Comment:
Shouldn't the entity be `CLIENTBUSINESSDETAIL` for consistency

Reply:
I used CLIENT in the handler because on frontend this object is attached on a client profile